core/src/main/java/io/opencensus/tags/TagContext.java

```diff
@@ -147,5 +158,17 @@ public Builder clear(TagKey key) {
     public TagContext build() {
       return new TagContext(tags);
     }
+
+    /**
+     * Enters the scope of code where the {@link TagContext} created from this builder is in the
+     * current context and returns an object that represents that scope. The scope is exited when
+     * the returned object is closed.
+     *
+     * @return an object that defines a scope where the {@code TagContext} created from this builder
+     *     is set to the current context.
+     */
+    public NonThrowingCloseable buildScoped() {
```

Comment:
Is this consistent with tracing? I thought the usage would be something like `Tagger.withTagContext(builder.build())`